pkg/controller/jivavolume/jivavolume_controller.go

```diff
@@ -209,10 +209,7 @@ func (r *ReconcileJivaVolume) bootstrapJiva(cr *jv.JivaVolume, reqLog logr.Logge
 
 // TODO: add logic to create disruption budget for replicas
 func createReplicaPodDisruptionBudget(r *ReconcileJivaVolume, cr *jv.JivaVolume, reqLog logr.Logger) error {
-	min, err := strconv.Atoi(cr.Spec.ReplicationFactor)
-	if err != nil {
-		return fmt.Errorf("failed to convert to int, err: %v", err)
-	}
+	min := cr.Spec.Policy.Target.ReplicationFactor
```

Comment:
It's good to add a prerequisite function to verify if required parameters exist in policy before bootstraping and return an error in case if something is missing. Jiva operator should be working independently, irrespective of jiva-csi or other projects which is creating and consuming JivaVolume CR. Adding these checks reduce the probability of bugs which is hard to catch sometimes. What do you say?

Reply:
The jiva-csi creates the JivaVolume CR with either the parameters provided in the policy or with default parameters. Are we considering the case where some user removes the parameter ?